Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)

```

Comment:
**Suggestion**: The loop increment i = i * 2 will cause an IndexOutOfBoundsException for arrays with more than 1 element. Use i += 2 for iterating through even indices, or revise the logic to correctly identify even numbers. <br> **Code Suggestion**: 
 ```
 -        for (int i = 0; i < arr.length; i = i * 2)
 +        for (int i = 0; i < arr.length; i += 2)
 ```